pages/annotation.html

```diff
@@ -14,32 +14,26 @@
         <link rel="stylesheet" href="../styles/annotation.css">
     </head>
     <body>
-        <div id="navbar" class="row">
-            <div id="logo" class="col-md-9"></div>
-            <div id="user" class="col-md-3"></div>
-        </div>
         <div id="swiper" class="row">
-            <div id="bad" class="col-md-3"></div>
-            <div class="col-md-6">
-                <div id="polaroid">
+            <div id="bad" class="col-md-3 hidden-xs hidden-sm"></div>
+            <div class="col-md-6 card-container">
+                <div id="polaroid" class="card">
```

Comment:
What does the card here do? Also I though you were going to change this to flex boxes?